doc/source/rllib/package_ref/index.rst

```diff
@@ -27,5 +27,6 @@ If you think there is anything missing, please open an issue on `Github`_.
    offline.rst
    execution.rst
    utils.rst
+   replay_buffers.rst
```

Comment:
I think we should move this here: @maxpumperla ?

<img width="290" alt="image" src="https://user-images.githubusercontent.com/5839854/171048848-3e6fff56-51e8-40d1-9e55-1431dbd2e07a.png">

Reply:
Got it! Moved.